.github/workflows/automatic-release.yml

```diff
@@ -0,0 +1,38 @@
+name: automatic-release
+
+on:
+    workflow_dispatch:
+        inputs:
+            release_type:
+                description: Release type
+                required: true
+                type: choice
+                options:
+                    - patch
+                    - minor
+                    - major
+
+jobs:
+    release:
+        name: Release
+        runs-on: ubuntu-22.04
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v4
+            - name: Setup Git
+              run: |
+                git config --local user.email "action@github.com"
+                git config --local user.name "GitHub Action"
+            - name: Setup Python
+              uses: actions/setup-python@v4
+              with:
+                python-version: '3.8'
```

Comment:
Should this be the lowest version of python we support? Whats the lowest supported python version out there?

Reply:
I'll be honest, I have no clue if there is any version issue at all here. For all I know it might be OK from 3.0 to 3.12, but I'm not familiar with any decent way to determine it either. I was just looking at the `README.rst` shield.